src/util.js

```diff
@@ -39,10 +40,15 @@ export function truncate_id(id) {
     return id.substr(0, 12);
 }
 
-export function localize_time(unix_timestamp) {
-    const locale = (cockpit.language == "en") ? dfnlocales.enUS : dfnlocales[cockpit.language.replace('_', '')];
-    return formatRelative(unix_timestamp * 1000, Date.now(), { locale });
-}
+// this supports formatted strings (via Date.parse) or raw timestamps
+export const RelativeTime = ({ time }) => {
+    if (!time)
+        return null;
```

Comment:
This added line is not executed by any test.

Reply:
Yeah, this is more like belt-and-suspenders. We directly plumb the podman API to this component, and at least in the past these fields could be null or undefined.